bolts/data/datamodules/common.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
+
+from typing_extensions import Literal
+
+__all__ = ["Stage"]
+
+Stage = Literal["fit", "validate", "test"]
```

Comment:
we use different terminology elsewhere (train, val, test) - we should consolidate

Reply:
We do, and incorrectly. I checked the lightning docs last night - what I have here is the correct set of options for Stage.